docs/serving/load-balancing/README.md

```diff
@@ -24,12 +24,13 @@ The Activator load balancing algorithm works as follows:
 
 For more information, see the documentation on [concurrency](../../serving/autoscaling/concurrency).
 
-## Configuring target burst capacity
+## Configuring target burst capacity behavior
 
 Target burst capacity is mainly responsible for determining whether the Activator is in the request path outside of scale-from-zero scenarios.
 
-Target burst capacity can be configured using a combination of the following parameters:
+Target burst capacity and its effect of the system can be configured using a combination of the following parameters:
```

Comment:
I don't believe that it's necessary to change this line. I don't think "and its effect of the system" adds anything here, it just makes me wonder "what effects is this referring to?" and we don't really explain how it works here. I'd put it back how it was unless there's some reason it has to change?